android/app/src/main/java/com/epfl/dedis/hbt/utility/json/JsonType.kt

```diff
@@ -1,5 +1,6 @@
 package com.epfl.dedis.hbt.utility.json
 
 enum class JsonType(val schemaPath: String) {
-    TRANSACTION("protocol/transaction.json")
+    TRANSACTION("protocol/transaction.json"),
```

Comment:
So why not renaming this a COMPLETE_TRANSACTION ?

Reply:
True, this has been done.